packages/react-components/src/templates/NewsDetailsPage.tsx

```diff
@@ -23,8 +23,8 @@ const richTextContainer = css({
 
 type NewsDetailsPageProps = Pick<
   NewsResponse,
-  'text' | 'title' | 'type' | 'created' | 'link' | 'linkText'
->;
+  'text' | 'title' | 'created' | 'link' | 'linkText'
+> & { type?: string };
```

Comment:
Rather than retyping could we use 

```
type NewsDetailsPageProps = Pick<
  NewsResponse,
  'text' | 'title' | 'type' | 'created' | 'link' | 'linkText'
> | Pick<
  TutorialResponse,
  'text' | 'title' | 'created' | 'link' | 'linkText'
>
```